src/Components/v2/Carousel.tsx

```diff
@@ -150,7 +155,7 @@ export class BaseCarousel extends React.Component<
 > {
   state = {
     currentSlideIndex: 0,
-    lastItemVisible: false,
+    lastItemVisible: true,
```

Comment:
Will this affect other places this component is used?

Reply:
Yes but this is better behavior. We should assume the carousel is completely visible by default and then only display the arrow if there's hidden content. This will prevent an arrow flashing.